test/metric.jl

```diff
@@ -62,6 +62,10 @@ function Manifolds.sharp!(::BaseManifold, v::FVector{Manifolds.TangentSpaceType}
 end
 
 @testset "Metrics" begin
+    @testset "Metric Basics" begin
+        #one for MetricManifold, one for Manifold & Metric
+        @test length(methods(is_default_metric)) == 2
```

Comment:
This should be `== 1`. This enforces that all logic is in the `dispatch` functions (which should be the case).

Reply:
Ah, OK, *these* two methods... It getting harder and harder to keep track of things.

Reply:
It's just this one that has two, because one with for the `MetricManifold` the other is for a separate pair of `Manifold` and `Metric`.